scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.DotNet.Cli.Build
     public class BuildVersion : Version
     {
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
-        public string VersionSuffix => $"{CommitCountString}";
+        public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
```

Comment:
This should not be here. The Release Suffix is already in the nupkg versions, so this doubles it up. We don't want to get nupkgs with -preview1-preview1-{version}.